features/admin.applications.v1/components/forms/share-application-form-updated.tsx

```diff
@@ -415,7 +415,10 @@ export const ApplicationShareFormUpdated: FunctionComponent<ApplicationShareForm
         if (isApplicationShareOperationStatusEnabled) {
             handleAsyncSharingNotification(shareType);
         }
-
+        if (shareType === ShareType.SHARE_SELECTED && selectedOrgIds.length === 0) {
+            noOrgSelected();
+            return;
+        }
```

Comment:
Let's move this early return conditionals to the top of the `handleApplicationSharing` method for better readability